futures-util/src/stream/catch_unwind.rs

```diff
@@ -52,20 +52,11 @@ impl<S> Stream for CatchUnwind<S>
                 self.state = CatchUnwindState::Stream(stream);
                 match poll {
                     Err(e) => Ok(Async::Ready(Some(Err(e)))),
-                    Ok(Async::NotReady) => Ok(Async::NotReady),
+                    Ok(Async::Pending) => Ok(Async::Pending),
                     Ok(Async::Ready(Some(r))) => Ok(Async::Ready(Some(Ok(r)))),
                     Ok(Async::Ready(None)) => Ok(Async::Ready(None)),
                 }
             }
         }
     }
 }
-
-impl<S: Stream> Stream for AssertUnwindSafe<S> {
```

Comment:
I think this didn't make it through to the other end?

Reply:
It had to get moved into `futures-core` for coherence: https://github.com/rust-lang-nursery/futures-rs/pull/734/files/304d0bc9d07ac3d7b5d95bf691b1031f641e290a#diff-c3412131fd6221fd427f2ed71eca84abR30